lib/provider/data_types.go

```diff
@@ -79,7 +79,13 @@ type Volume struct {
 	// LunID the lun of volume
 	LunID string `json:"lunId,omitempty"`
 
-	// IscsiTargetIPAddresses list of target IP addresses for iscsi. Applicable for Iscsi block storage only
+	// TargetIPAddresses list of target IP addresses for iscsi
+	TargetIPAddresses []string `json:"IscsiTargetIpAddresses,omitempty"`
```

Comment:
`TargetIPAddresses` is not required as we already have `IscsiTargetIPAddresses`   just name different